src/Components/reusables/DiveLogImageCarousel.tsx

```diff
@@ -29,23 +29,21 @@ enum Directions {
   left = 'left',
 }
 
-interface Images {
-  source: ImageSourcePropType;
+interface Image {
+  uri: string;
+  type?: string;
+  name?: string;
 }
 
-const defaultImages: Images[] = [
+const defaultImages: Image[] = [
   {
-    source: DivingPlaceholder,
+    uri: DivingPlaceholder,
```

Comment:
is it okay to change this?

Reply:
The naming of the default image source is also deliberate because it is an image source, rather than a link to en external asset.

Reply:
i updated it to be an external asset so that they can be consistent